evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -173,6 +174,9 @@ def test_fast_permutation_importance_matches_sklearn_output(mock_supports_fast_i
 class PipelineWithDimReduction(BinaryClassificationPipeline):
```

Comment:
Could simplify this to not use classes, but I like that the name makes it clear without overstuffing the parameterize field so I left as is :)